README.md

```diff
@@ -74,37 +74,37 @@ It also requires PHP's `zip` extension installed and enabled.
 # Copyright
 This project was developed by :
 
-Neontribe Ltd (registered in England and Wales #06165574) 
+Neontribe Ltd (registered in England and Wales #06165574)
 
 Under contract for
 
-Alexandra Rose Charity (registered in England and Wales #00279157) 
+Alexandra Rose Charity (registered in England and Wales #00279157)
 
-As such, unless otherwise specified in the appropriate component source, associated file or compiled asset, files in this project repository are Copyright &copy; (2020), Alexandra Rose Charity. All rights reserved.
+As such, unless otherwise specified in the appropriate component source, associated file or compiled asset, files in this project repository are Copyright &copy; (2021), Alexandra Rose Charity. All rights reserved.
 
 If you wish to discuss copyright or licensing issues, please contact:
 
 Alexandra Rose Charity
 
-c/o Wise & Co, 
-Wey Court West, 
-Union Road, 
-Farnham, 
-Surrey, 
+c/o Wise & Co,
+Wey Court West,
+Union Road,
+Farnham,
+Surrey,
 England,
 GU9 7PT
 
 # Licensing and use of Third Party Applications
 These are the languages and packages used to create ARCV Service and where available the licences associated with them.
 
-ARCV Service 1.9
+ARCV Service 1.12
 
 Programming Language - PHP\
 Framework - Laravel https://github.com/laravel/laravel \
 Licence - The Laravel framework is open-sourced software licensed under the MIT license.
 
 Third Party Packages
-- https://github.com/barryvdh/laravel-cors MIT Licence https://github.com/barryvdh/laravel-cors/blob/master/LICENSE 
+- https://github.com/barryvdh/laravel-cors MIT Licence https://github.com/barryvdh/laravel-cors/blob/master/LICENSE
```

Comment:
aha! this is no longer barryvdh/laravel-cors, he moved it to https://github.com/fruitcake/laravel-cors

Reply:
done here https://trello.com/c/yvfPnN8H/1671-update-third-party-package-licensing